### PR TITLE
Feat :Display labels & customize Markers in ScatterSeries (PCA)

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium/react-plot/issues/151

### CODE REVIEW

File: stories/examples/pca.stories.tsx [hunk 1]
```diff
@@ -1,76 +1,200 @@
...
+        point1: { x: x1, y: y1 },
+        point2: { x: x2, y: y2 },
+      } = majorAxis;
+      return {
+        x1,
+        y1,
+        x2,
+        y2,
+        ...position,
+        width: rMinor,
```
Comment (lpatiny) [inline stories/examples/pca.stories.tsx:76]:
```suggestion
        width: rMinor * 2,
```

**Comment by lpatiny** [comment]
It works as I expected (small mistake with the radius used for width).

**Comment by lpatiny** [comment]
@wadjih-bencheikh18 Please also check the related issue: https://github.com/zakodium/react-plot/issues/297

File: src/components/ScatterSeries.tsx [hunk 2]
```diff
@@ -126,18 +142,20 @@ function ScatterSeriesRender({
...
+      const Marker = markersComps[functionalShape(markerShape, point, i, data)];
+      const label = functionalLabel(pointLabel, point, i, data);
+      const labelStyle = functionalStyle({}, pointLabelStyle, point, i, data);
       return (
         <g // eslint-disable-next-line react/no-array-index-key
           key={`markers-${i}`}
           transform={`translate(${xScale(point.x)}, ${yScale(point.y)})`}
         >
           <Marker size={markerSize} style={{ stroke: style.fill, ...style }} />
+          <text style={labelStyle}>{label}</text>
```
Comment (targos) [inline src/components/ScatterSeries.tsx:158]: Maybe we should only render the `text` element if `label` is not empty?

**Pull request merged**